builds.expected

```diff
@@ -9027,8 +9027,71 @@ windows-mingw-20H2/amd64
 	ENTRYPOINT [ "ocaml-env", "exec", "--64", "--" ]
 	CMD [ "cmd.exe" ]
 	COPY [ "Dockerfile", "/Dockerfile.ocaml" ]
-ocurrent/opam-staging:windows-mingw-1809-ocaml-4.12-amd64, ocurrent/opam-staging:windows-mingw-2004-ocaml-4.12-amd64, ocurrent/opam-staging:windows-mingw-20H2-ocaml-4.12-amd64 -> ocaml/opam:windows-mingw
-ocurrent/opam-staging:windows-mingw-1809-ocaml-4.12-amd64, ocurrent/opam-staging:windows-mingw-2004-ocaml-4.12-amd64, ocurrent/opam-staging:windows-mingw-20H2-ocaml-4.12-amd64 -> ocaml/opam:windows-mingw-ocaml-4.12
+windows-mingw-ltsc2022/amd64
+	# escape=`
+	FROM mcr.microsoft.com/windows:ltsc2022-KB5005039 as winget-builder
```

Comment:
This isn't right?

Reply:
Good catch, thanks! fixed with the latest changes to ocaml-dockerfile.